SPEC.md

```diff
@@ -98,61 +98,104 @@ A build tool is something which reads one or more input artifacts and writes one
 ## Specifications
 ### Artifact ID
 
-GitBOM uses the git object id (as a blob) for an artifact as its artifact ID.
+A hash function over the []byte of an artifact satisfies the canonical, unique, immutable requirements for an artifact identifier.
```

Comment:
Nitpick, but the `[]byte` Go-ism may be better replaced by "byte array" or some other similarly-generic term. The specific syntax is _pretty-intelligible_ as-is, but may still be a slight barrier to understanding depending on audience background.

Reply:
@alilleybrinker Suggestions on how to handle the use of []byte further up in artifact equivalency?

```
Two artifacts are equivalent if and only if `[]byte(artifact1) == []byte(artifact2)`
```

Reply:
Hmm... I take it the point here is that two articles are equivalent only if their bit representation is equal, so perhaps say that directly?